app/src/main/java/cm/aptoide/pt/view/BaseActivity.java

```diff
@@ -15,6 +15,8 @@
 
   @Override protected void onCreate(@Nullable Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
```

Comment:
not sure if it is possible to inject stuff here bt it'd be worth the try

Reply:
It is. Done